src/main/python/keyboard_to_nt.py

```
@@ -1,33 +1,143 @@
+                keys_dict.pop(key, None)
+
+
+def on_action(event: keyboard.KeyboardEvent):
+    if event is None or event.name is None or event.name == "/":
+        return
+
+    key = ("numpad" + event.name) if event.is_keypad else event.name.lower()
+    is_pressed = event.event_type == keyboard.KEY_DOWN
+
+    if is_pressed:
+        sent = put_boolean_safe(key, True)
```

Ruff (0.15.5)

[error] Boolean positional value in function call

(FBT003)